.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: KBase Auth2 Client tests
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+jobs:
+
+  auth2_java_client_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - java: '8'
+          - java: '11'
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
uuuuggghhh, since both you and Sijie are asking, FINE, jeez. 

And yeah, codecov 4 break other repos for me. I think there's at minimum some backwards incompatible changes you have to account for but I haven't looked into it thoroughly yet.

Reply:
I've found some long github threads of people complaining about the same things I've seen, so I think it's on their end. Swapping back to v3 fixed my problems right away.

Reply:
> Swapping back to v3 fixed my problems right away.

Same